docs/content/guide/echo/typescript/subscriptions.md

```diff
@@ -0,0 +1,44 @@
+---
+order: 3
+---
+
+# Subscriptions
+
+Use subscriptions to listen for changes within a [space](./README.md).
```

Comment:
More prose wouldn't hurt here if people have suggestions on the details of how they work.

Reply:
...in particular, now that this page covers two different types of subscriptions, some writing on how they differ could be useful.